packages/ember-htmlbars/tests/compat/make_bound_helper_test.js

```diff
@@ -101,7 +101,7 @@ QUnit.test('should update bound helpers in a subexpression when properties chang
   ignoreDeprecation(function() {
     view = EmberView.create({
       controller: { prop: 'isThing' },
-      template: compile('<div {{bind-attr data-foo=(dasherize prop)}}>{{prop}}</div>')
```

Comment:
@mmun I'm considering that putting a mustache in element space results in different hooks being called than putting it in the attr value. With helpers those differences were a constant pain, and I'm presuming this test is specifically for testing a binding in element space.